app/views/reviewRegistration.html

```diff
@@ -106,11 +106,14 @@ <h2 class="page-title border" translate>Registration Review</h2>
           ng-init="expandRegistrant[r.id] = regValidate[r.id].length > 0"
         >
           <tr>
-            <td ng-click="expandRegistrant[r.id] = !expandRegistrant[r.id]">
-              <i
-                class="fa"
+            <td>
+              <button
+                aria-label="Show registrant details"
+                ng-click="expandRegistrant[r.id] = !expandRegistrant[r.id]"
+                class="fa btn-unstyled"
```

Comment:
It doesn't look like the `.btn-unstyled` class does anything. Can you add a CSS rule to remove the border and background from `.btn-unstyled` and then remove the inline styles?